src/playsim/actor.h

```diff
@@ -413,6 +413,8 @@ enum ActorFlag8
 	MF8_RECREATELIGHTS	= 0x00000100,	// Internal flag that signifies that the light attachments need to be recreated at the
 	MF8_STOPRAILS		= 0x00000200,	// [MC] Prevent rails from going further if an actor has this flag.
 	MF8_ABSVIEWANGLES	= 0x00000400,	// [MC] By default view angle/pitch/roll is an offset. This will make it absolute instead.
+	MF8_VIEWPOSNOANGLES	= 0x00000800,	// [MC] View position takes no angles into account
```

Comment:
Isnt having VIEWPOSIGNOREYAW(PITCH/ROLL) here would be more suitable? I mean three flags for each camera rotation axis.